js/components/thumbnailCarousel.js

```diff
@@ -27,19 +27,22 @@ var ThumbnailCarousel = React.createClass({
                    centerThumbnailHeight: carousel.clientHeight});
   },
 
+  componentWillReceiveProps: function(nextProps) {
```

Comment:
This is a workaround for conflict with test code, I was telling you about it on Friday.  This method is called by real code, but it is not called by test code.  So, after it is set to false, real code will be just comparing properties, i.e. do the right job.